bitcoin/test/miniscript_tests.cpp

```diff
@@ -498,6 +499,7 @@ void Test(const std::string& ms, const std::string& hexscript, int mode, int ops
         if (hexscript != "?") BOOST_CHECK_MESSAGE(HexStr(computed_script) == hexscript, "Script mismatch: " + ms + " (" + HexStr(computed_script) + " vs " + hexscript + ")");
         BOOST_CHECK_MESSAGE(node->IsNonMalleable() == !!(mode & TESTMODE_NONMAL), "Malleability mismatch: " + ms);
         BOOST_CHECK_MESSAGE(node->NeedsSignature() == !!(mode & TESTMODE_NEEDSIG), "Signature necessity mismatch: " + ms);
+        BOOST_CHECK_MESSAGE(node->CheckTimeLocksMix() == !(mode & TESTMODE_TIMELOCKMIX), "Timelock mix mismatch: " + ms);
```

Comment:
Missing an exclamation mark :)

Reply:
Only one exclamation is required since the ChecktimelocksMix returns if there is a timelock mix. So, the single exclamation mark is correct

Reply:
Gotcha, i was confused. You are testing for the presence of locktime mix!